website/src/data/users.js

```diff
@@ -827,6 +827,14 @@ const users = [
     source: 'https://github.com/Wisdom/dev-docs',
     tags: ['design', 'large'],
   },
+  {
+    title: '前端进阶导航（2021版）',
```

Comment:
is it really required to add "2021" here? looks like your site will remain actual even when we'll be in 2022 no?